src/include/cart/api.h

```diff
@@ -1777,6 +1778,18 @@ crt_rank_self_set(d_rank_t rank);
 int
 crt_rank_uri_get(crt_group_t *grp, d_rank_t rank, int tag, char **uri);
 
+/**
+ * Get rank SWIM state.
+ *
+ * \param[in] rank              Rank to get SWIM state for
+ *
```

Comment:
doxygen missing for other params of the function

Reply:
done